src/styles/Theme.js

```diff
@@ -0,0 +1,29 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+export const theme = createMuiTheme({
+  palette: {
+    blue: {
+      main: '#a1dffb',
+    },
+    red: {
+      main: '#c33124',
+    },
+    orange: {
+      main: '#e8a628',
+    },
+    salmon: {
+      main: '#f98365',
+    },
+    yellow: {
+      main: '#F9de59',
+    },
+  },
+});
+
+console.log('theme file ', theme);
```

Comment:
maybe can get rid of this and the commented out code below. 👍

Reply:
There is one issue with color contrast - if you enter an invalid token on the welcome screen, the WAVE plugin registers poor contrast between the form's yellow background and the red error message. Due to my internet issues, @DeeDowns has kindly offered to resolve this error.